### DISCUSSION TITLE
Cannot register with error.

### DISCUSSION THREAD

**Discussion by oha-4** [community/community#40895] [GitHub Education] [answered]
The following error occurred even after filling in all profiles.
I don't want to reveal my name to the public, so I temporarily set it to my real name when I registered, but the same error.
I am Japanese, but is it because my certificate is written in Kanji characters?

---
Your application cannot be reviewed until you fix the following:
- Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?,You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.
---

**Comment by ossanna16** [comment]
Hi there @sheat-git and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by oha-4** [reply]
Sorry. I solved the problem and forgot to answer.

By the way, do you plan to support certificates written in languages other than English?

**Comment by oha-4** [comment, marked as answer]
I solved this problem by using a certificate written in English.

**Reply by spike-h** [reply]
What type of certificate did you upload? All of my photos are in English but it keeps giving me this error

**Reply by somebody-someone** [reply]
Me too. All my proof upload are written in English, and I fill all my profile, but the problem still happens.

**Reply by samus-aran** [reply]
Hi @spike-h & @somebody-someone 👋 

We're happy you're here! You are more likely to get a useful response if you are explicit about what your reasons rejection are (a screenshot may help) and any additional info you think is relevant. Giving a few more details might help someone give you a nudge in the right direction. 👍

**Reply by spike-h** [reply]
[deleted the picture due to some personal information in it]

I uploaded a picture to prove my student status but it keeps saying that it can't preprocess my application. I've added a bio and set a location, but it still won't work. This is also the second picture I've tried.

**Reply by samus-aran** [reply]
Thanks @spike-h 

Let's have a look at the rejection reasons and some other tips to be aware of when applying.

1. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. Do you have one? If you don't have an Academic email do try and acquire evidence that is positioned higher rather than lower in that option list.

2. Ensure your GitHub profile corresponds to the evidence you submit. The verification process checks your GitHub username, email and profile name (as well as aspects) - is there any reference to these in your evidence? 

Hope this helps. 👍

**Reply by spike-h** [reply]
Thanks for the reply, sending in a form of evidence closer to the top (a transcript) allowed my application to pass through

**Reply by samus-aran** [reply]
Awesome @spike-h 🎉  I'm super glad to hear that 👍
